src/main/java/it/gov/pagopa/cgn/portal/converter/backoffice/approved/BackofficeApprovedDiscountConverter.java

```diff
@@ -22,17 +22,20 @@
 
 
     private static final Map<ProductCategoryEnum, ProductCategory> productEnumMaps = new EnumMap<>(ProductCategoryEnum.class);
+
     static {
-        productEnumMaps.put(ProductCategoryEnum.ENTERTAINMENT, ProductCategory.ENTERTAINMENT);
-        productEnumMaps.put(ProductCategoryEnum.TRAVELLING, ProductCategory.TRAVELLING);
-        productEnumMaps.put(ProductCategoryEnum.FOOD_DRINK, ProductCategory.FOODDRINK);
-        productEnumMaps.put(ProductCategoryEnum.SERVICES, ProductCategory.SERVICES);
+        productEnumMaps.put(ProductCategoryEnum.BANKING_SERVICES, ProductCategory.BANKINGSERVICES);
+        productEnumMaps.put(ProductCategoryEnum.CULTURE_AND_ENTERTAINMENT, ProductCategory.CULTUREANDENTERTAINMENT);
+        productEnumMaps.put(ProductCategoryEnum.HEALTH, ProductCategory.HEALTH);
+        productEnumMaps.put(ProductCategoryEnum.HOME, ProductCategory.HOME);
+        productEnumMaps.put(ProductCategoryEnum.JOB_OFFERS, ProductCategory.JOBOFFERS);
         productEnumMaps.put(ProductCategoryEnum.LEARNING, ProductCategory.LEARNING);
-        productEnumMaps.put(ProductCategoryEnum.HOTELS, ProductCategory.HOTELS);
         productEnumMaps.put(ProductCategoryEnum.SPORTS, ProductCategory.SPORTS);
-        productEnumMaps.put(ProductCategoryEnum.HEALTH, ProductCategory.HEALTH);
-        productEnumMaps.put(ProductCategoryEnum.SHOPPING, ProductCategory.SHOPPING);
+        productEnumMaps.put(ProductCategoryEnum.SUSTAINABLE_MOBILITY, ProductCategory.SUSTAINABLEMOBILITY);
+        productEnumMaps.put(ProductCategoryEnum.TELEPHONY_AND_INTERNET, ProductCategory.TELEPHONYANDINTERNET);
```

Comment:
`PHONE_AND_INTERNET`?

Reply:
Telephony is more like telephony than phone. By the way, it's a name, do you really think PHONE is better than TELEPHONY? From https://context.reverso.net/traduzione/italiano-inglese/telefonia+e+internet it seems the same.